universal-login-react/src/ui/Notifications/ConnectionNotification.tsx

```diff
@@ -17,38 +22,81 @@ interface ConnectNotificationProps {
 export const ConnectionNotification = ({deployedWallet, devicesBasePath, className}: ConnectNotificationProps) => {
   const [notifications, setNotifications] = useState([] as Notification[]);
   const [showTitle, setShowTitle] = useState(true);
+  const [gasParameters, setGasParameters] = useState<GasParameters | undefined>(undefined);
+  const [publicKey, setPublicKey] = useState<string | undefined>(undefined);
+  const {progressBar, showProgressBar} = useProgressBar();
   useEffect(() => deployedWallet.subscribeAuthorisations(setNotifications), []);
 
   const history = useHistory();
 
-  const onDenyButtonClick = () => {
-    deployedWallet.denyRequests();
+  const onDenyButtonClick = async () => {
+    await deployedWallet.denyRequests();
     history.goBack();
   };
 
+  const onCancelClick = async () => {
+    await deployedWallet.denyRequests();
```

Comment:
Look really similar to `onDenyButtonClick`. Maybe this one shouldn't deny requests

Reply:
Is seems to be refered as a "reject" button:
https://github.com/UniversalLogin/UniversalLoginSDK/blob/9dc642b18798d6c4e54ca00f8e7b94dd1e78ca38/universal-login-react/src/ui/Notifications/EmojiForm.tsx#L66-L70

Shall we rename this to `onRejectClick`? Also then it would make sense that "reject" denies the requests